docs/source/installation/login.md

```diff
@@ -94,46 +104,47 @@ qhub keycloak -c qhub-config.yaml adduser <username> <password>
 
 This will create a user `<username>` with the initial password provided. Omit the password completely if you are using GitHub or Auth0.
 
-> NOTE: This will also add the user to 'analyst' group.
+> NOTE: This will also add the user to the 'analyst' group.
 
 ## Login to QHub
 
-Your new user can now log into the main QHub dashboard (not Keycloak's admin console).
+Your new user can now log in to the main QHub dashboard (not Keycloak's admin console).
 
 Visit <https://myqhubsite.com/> (or whatever domain you have chosen for your QHub).
 
-Click 'Sign in with Keycloak'.
+Click 'Sign in with Keycloak'. This will take you to the login form:
```

Comment:
I don't love the two-step login, even if it's used at places as preeminent as Gmail. I wonder if the button should say "Sign in with Keycloak" or just "Sign In".

Reply:
well, that's customizable, but I don't see a good enough reason for the change. But is open for discussion :slightly_smiling_face: